SqlInstall.ps1

```diff
@@ -93,6 +96,18 @@
         Stop-Function -Message "You need to specify a SQL Server Version to run this function." -Continue -EnableException $EnableException
     }
 
+    If($Null -eq $Edition -or $Edition -eq '')
+    {
+        $Edition = "Express"
```

Comment:
You can actually take care of this in the parameters - so do `$Edition = "Express"` and it'll auto fill express if they didn't pass an Edition 👍